src/User32.Desktop/User32.cs

```diff
@@ -647,13 +1103,71 @@ public static partial class User32
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool GetClassInfoEx(IntPtr hInstance, string lpClassName, ref WNDCLASSEX lpWndClass);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
+        public static extern unsafe bool GetMonitorInfo(
+            IntPtr hMonitor,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Optional)] MONITORINFO* lpmi);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
+        public static extern unsafe bool GetMonitorInfoEx(
+            IntPtr hMonitor,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Optional)] MONITORINFOEX* lpmi);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int GetSystemMetrics(SystemMetric smIndex);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool GetWindowInfo(IntPtr hwnd, ref WINDOWINFO pwi);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int MapWindowPoints(IntPtr hWndFrom, IntPtr hWndTo, ref RECT rect, [MarshalAs(UnmanagedType.U4)] int cPoints);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr MonitorFromPoint(POINT pt, MonitorOptions dwFlags);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr MonitorFromPoint(POINT point, int flags);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr MonitorFromRect(ref RECT lprc, MonitorOptions dwFlags);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr MonitorFromWindow(IntPtr hwnd, MonitorOptions dwFlags);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool SystemParametersInfo(SPI uiAction, uint uiParam, IntPtr pvParam, SPIF fWinIni);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int QueryDisplayConfig(uint Flags, ref uint pNumPathArrayElements, ref DISPLAYCONFIG_PATH_INFO pPathInfoArray, ref uint pNumModeInfoArrayElements, ref DISPLAYCONFIG_MODE_INFO pModeInfoArray, ref DISPLAYCONFIG_TOPOLOGY_ID pCurrentTopologyId);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern ushort RegisterClass(ref WNDCLASS lpWndClass);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.U2)]
+        public static extern short RegisterClassEx(ref WNDCLASSEX lpwcx);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int GetMessage(out MSG lpMsg, IntPtr hWnd, uint wMsgFilterMin, uint wMsgFilterMax);
```

Comment:
Also we already have a declaration for GetMessage with correct pointer types here : https://github.com/AArnott/pinvoke/blob/master/src/User32.Desktop/User32.cs#L1438 Why add one more ?